libs/blocks/quiz/quiz.css

```diff
@@ -1,980 +1,384 @@
-  max-width: 100%;
-  margin: 0 0 16px;
-}
-
-.quiz-container .consonant-CardsGrid--5up .card div div {
-  text-align: right;
+.quiz-option.selected .quiz-option-title {
+  color: var(--link-hover-color-dark);
+  font-weight: bold;
 }
 
-.quiz-options-container .quiz-option.disabled {
-  cursor: not-allowed;
-  pointer-events: none;
-  outline: none;
+.quiz-option.selected .quiz-option-text {
+  color: var(--link-hover-color-dark);
+  font-weight: bold;
 }
 
-.card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option.disabled {
-  cursor: not-allowed;
-  pointer-events: none;
-  outline: none;
+.quiz-option.disabled {
   background-color: #fafafacc;
   box-shadow: 10px 10px 6px 0 #00000029;
+  cursor: not-allowed;
+  outline: none;
 }
 
-.card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option.disabled .consonant-OneHalfCard-image,
-.card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option.disabled .consonant-OneHalfCard-inner {
+.quiz-option.disabled .quiz-option-icon,
+.quiz-option.disabled .quiz-option-image,
+.quiz-option.disabled .quiz-option-text-container{
   opacity: 0.5;
 }
 
-.quiz-container .consonant-OneHalfCard-inner {
-  background-color: transparent;
-  border-radius: 10px;
-}
-
-.quiz-container .consonant-OneHalfCard-image {
-  margin-right: 16px;
-}
-
-.quiz-container .card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option:hover {
-  background-color: var(--color-white);
-}
-
-.quiz-container .consonant-OneHalfCard-inner {
-  flex-direction: row;
-}  
-
-.quiz-container .consonant-OneHalfCard-cover {
-  width: 45px;
-  height: 45px;
-  object-fit: cover;
-  border-radius: 0.5rem;
-  margin-right: 1rem;
+.quiz-button-container {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin: 20px 0 24px;
 }
 
-.quiz-container .card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option .icon {
-  width: 4rem;
-  height: 4rem;
+.quiz-button {
+  background: var(--color-white);
+  border: 2px solid var(--color-white);
+  border-radius: 30px;
+  cursor: pointer;
+  font-size: 14px;
+  font-weight: 700;
+  min-height: 32px;
+  min-width: 100%;
+  padding: 13px 86px;
 }
 
-.show {
-  display: block; 
+.quiz-button-label {
+  color: var(--link-hover-color-dark);
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 22px;
 }
 
-.hide {
-  display: none;
+.quiz-button[disabled] {
+  background: transparent;
+  cursor: not-allowed;
 }
 
-.quiz-page .footer-info {
-  margin-top: 12px;
+.quiz-button[disabled] .quiz-button-label{
+  color: var(--color-white);
 }
 
-/* Step Indicator CSS */
 .dot-indicators {
-  width: 300px;
-  top: 25px;
-  z-index: 100;
-  display: flex;
   align-items: center;
-  justify-content: space-between;
-}
-
-.dot-indicators:first-of-type {
-  display: none;
-}
-
-.dot-indicators:last-of-type {
   display: flex;
-  visibility: visible;
+  justify-content: space-between;
+  width: 300px;
+  z-index: 15;
 }
 
 .dot {
```

Comment:
I'd argue that `.dot` might be a bit too generic, maybe it could be scoped to the quiz block

Reply:
renamed .dot-indicators to quiz-step-container and .dot to .quiz-step. This should guarantee that we don't have any potential collisions